ui/app/src/views/ViewDashboardList.tsx

```diff
@@ -31,9 +31,34 @@ import { ErrorAlert, ErrorBoundary } from '@perses-dev/components';
 import { ChevronDown, FolderPound } from 'mdi-material-ui';
 import { useDashboardList } from '../model/dashboard-client';
 
+export interface DashboardListProperties {
+  dashboardList: DashboardResource[];
+}
+
+export function DashboardList(props: DashboardListProperties) {
+  const navigate = useNavigate();
+  return (
+    <List>
+      {props.dashboardList.map((dashboard, i) => {
+        return (
+          <Box sx={{ backgroundColor: (theme) => theme.palette.primary.main + '10' }} key={dashboard.metadata.name}>
```

Comment:
Could a <ListItem> be used here instead of Box?

Reply:
ah yeah that would make more sense. I will try it. Thanks for your eyes ! 🙏